server/user.go

```diff
@@ -28,10 +29,12 @@ type User struct {
 type Connection struct {
 	jira.User
 	PluginVersion      string
-	Oauth1AccessToken  string `json:",omitempty"`
-	Oauth1AccessSecret string `json:",omitempty"`
+	Oauth1AccessToken  string        `json:",omitempty"`
+	Oauth1AccessSecret string        `json:",omitempty"`
+	OAuth2Token        *oauth2.Token `json:",omitempty"`
```

Comment:
just a question is `,` required here?

Reply:
I think yes. THe perm before `,` is the json name of the field